content/en/docs/ops/integrations/spire/index.md

```diff
@@ -286,26 +258,13 @@ Note that `SPIRE Controller Manager` is used in the [quick start](#option-1-quic
     {{< text syntax=bash snip_id=none >}}
     $ kubectl get pods -n istio-system
     NAME                                    READY   STATUS    RESTARTS   AGE
-    istio-ingressgateway-5b45864fd4-lgrxs   0/1     Running   0          20s
-    istiod-989f54d9c-sg7sn                  1/1     Running   0          25s
+    istio-ingressgateway-5b45864fd4-lgrxs   1/1     Running   0          17s
+    istiod-989f54d9c-sg7sn                  1/1     Running   0          23s
     {{< /text >}}
 
-    The Ingress-gateway pod and data plane containers will only reach `Ready` if a corresponding registration entry is created for them on the SPIRE Server. Then,
-    Envoy will be able to fetch cryptographic identities from SPIRE.
-    See [Register workloads](#register-workloads) to register entries for services in your mesh.
-
-The Istio configuration shares the `spiffe-csi-driver` with the Ingress Gateway and the sidecars that are going to be injected on workload pods,
-granting them access to the SPIRE Agent's UNIX Domain Socket.
-
-This configuration also adds an initContainer to the gateway that will wait for SPIRE to create the UNIX Domain Socket before starting the istio-proxy. If the SPIRE agent is not ready or has not been properly configured with the same socket path, the Ingress Gateway initContainer will wait forever.
-
-## Register workloads
-
-This section describes the options available for registering workloads in a SPIRE Server.
-
-### Option 1: Registration using the SPIRE Controller Manager
+    The Ingress-gateway pod is `Ready` since the corresponding registration entry is automatically created for it on the SPIRE Server. Envoy is able to fetch cryptographic identities from SPIRE.
```

Comment:
```suggestion
    The Ingress Gateway pod is `Ready` since the corresponding registration entry is automatically created for it on the SPIRE Server. Envoy is able to fetch cryptographic identities from SPIRE.
```